src/components/Header/MobileNavigation.tsx

```diff
@@ -87,7 +87,11 @@ const MobileNavigation: React.FC<
             >
               {stringGetter({ key: STRING_KEYS.DASHBOARD })}
             </Link>
-            <Link role="button" tabIndex={0} onClick={() => {}}>
+            <Link
+              role="button"
+              tabIndex={0}
+              onClick={() => window.open(ExternalLink.Forums, '_blank')}
+            >
               {stringGetter({ key: STRING_KEYS.FORUMS })}
               <LinkOutIcon />
             </Link>
```

Comment:
Is there a way to make this an `<a>` under the hood? `onClick` to open a link is quite bad for accessibility

Reply:
Is there a particular reason for this? Yes, we lose selectors like `:visited` and `:active`, but in terms of accessibility, every screen reader I've tested is able to handle these inputs, as long as `role` is specified. We enforce both `role` and tabIndex` with eslint rules.

Reply:
Yes, but that doesn't allow basic things like context menu on right click/long press, Ctrl-click to open in new tab, URL preview on hover, etc.

Reply:
I understand that for desktop links, this is the mobile nav though. I don't mind changing it, just curious what the benefits are on mobile.

Reply:
On iOS at least, you can long press/force press to get a preview/context menu. Or maybe the user is using an iPad on split screen with a Bluetooth keyboard and Apple Pencil. You never know, so best not to assume ;)